### DISCUSSION TITLE
Corrupt TDB File for Master

### DISCUSSION THREAD

**Discussion by Dragonwolf-Dominus** [TrinityCore/TrinityCore#25947] [Q&A] [answered]
Hello all.
I'm new here.
I try to install a new Shadowlands server, and to understand how it work, if it is the same as the official game, and how you do to do the same if it is... well, that's why I try to install it.
I will try too to change MySQL database by my old installed and constantly running SQL Server since I have used to manage this DB more than MySQL. (Will be a little tricky since I'm a .Net professional that never learned C++ (So I learn it at the same time).

I'm posting here because it seems that the TDB file TDB 837.20101 in your repository is corrupted and can't be opened.
I tryied multiple times to download it and open it with Winzip, WinRar and 7Zip... but each time it's saying the same, that zip is not in a well done format.
As you say in your documentation to not use another file for the first time init, it's a little stopped.

Just if you can verify if the archive is ok, maybe I'm wrong, but I don't think so.

Have a good day all.

**Comment by funjoker** [comment]
Works for me everytime i download it

**Comment by Aokromes** [comment]
i vote he is downloading the html instead the compresed file.

**Comment by ghost** [comment]
Have not tested the master TDB file yet, only compiling 3.3.5 on my own Windows computer. (Hang on, I will just test how it works to unzip/unRAR/un-7zip TDB_full_837.20101_2020_10_20.7z)

**Comment by ghost** [comment]
No issues, neither during download nor during extraction of the files `TDB_full_hotfixes_837.20101_2020_10_20.sql` + `TDB_full_world_837.20101_2020_10_20.sql`.
Make sure you download https://github.com/TrinityCore/TrinityCore/releases/download/TDB837.20101/TDB_full_837.20101_2020_10_20.7z by clicking the link (not right-clicking and selecting "download" from the menu).

**Comment by ghost** [comment]
Also, we don't know if you have got any web browser plugins affecting how things are downloaded or whether your unzip tools are modified or not.

**Comment by Dragonwolf-Dominus** [comment]
I use Edge, without any plugins or addons.
I tried this link (https://github.com/TrinityCore/TrinityCore/releases/download/TDB837.20101/TDB_full_837.20101_2020_10_20.7z), probably the same I used before, and it fails too.
Zip is downloaded, but when I tried to extrats files, they are extracted but empty, and 7zip show a window saying compression format is not a valid one.
I don't think it will change anything, but I will try to download it with another browser.
I'll come back to say you if it works.

**Reply by Dragonwolf-Dominus** [reply]
I tried with Firefox, and it's the same.
This is the message from WinRAR :
!   C:\TDB_full_837.20101_2020_10_20.7z: Méthode inconnue dans TDB_full_hotfixes_837.20101_2020_10_20.sql
!   C:\TDB_full_837.20101_2020_10_20.7z: Méthode inconnue dans TDB_full_world_837.20101_2020_10_20.sql
!   C:\TDB_full_837.20101_2020_10_20.7z: Erreur - l'opération a échoué

**Comment by Shauren** [comment, marked as answer]
Use 7zip to extract, not WinRAR

**Reply by Dragonwolf-Dominus** [reply]
As I said in the first post, I try with both Winzip, WinRar and 7Zip.
I just indicate WinRar alert because it is the easyest to copy.

I will try to update my 7Zip. 
Maybe it is a little old, but I'm not sure compression has evolved too much in less than 5 years.
I'll be back to say if it is better.

**Comment by Dragonwolf-Dominus** [comment]
Well, sorry all for disturbing....
I'm feeling a little stupid...
I have updated my 7Zip to the last version (2019-02-21), and it works.....
So compression methods are still in evolution...
So if anyone has the same problem, think to update your tools before.

Thanks all for the help.

**Comment by ghost** [comment]
Thank you for the feedback. Glad you were able to solve your issue, as well as discovering what caused the issue to happen in the first place.
